casr/src/bin/casr-dojo.rs

```diff
@@ -552,6 +560,174 @@ fn get_report_description(
     d
 }
 
+fn parse_env_var(s: &str) -> Option<&str> {
```

Comment:
Maybe we can hardcore exactly one supported environment variable `CASR_MAIL_PASSWORD`?